### DISCUSSION TITLE
how to make ui align correctly

### DISCUSSION THREAD

**Discussion by danielgoldin** [emacs-citar/citar#832] [Q&A]
I am a non-programmer using citar for academic papers. I can't seem to get the citar ui to align corectly. I'm running emacs 19. Here are my settings:

(use-package citar
  :no-require
  :custom
  (org-cite-global-bibliography '("~/org-roam/bibliography/refernces.bib"))
  (org-cite-insert-processor 'citar)
  (org-cite-follow-processor 'citar)
  (org-cite-activate-processor 'citar)
  (citar-bibliography org-cite-global-bibliography)
  ;; optional: org-cite-insert is also bound to C-c C-x C-@
  :bind
  (:map org-mode-map :package org ("C-c b" . #'org-cite-insert))
  :hook
  (LaTeX-mode . citar-capf-setup)
  (org-mode . citar-capf-setup))

Here's what my ui looks like:

![bib](https://github.com/emacs-citar/citar/assets/163761153/2b705f00-81b7-4d50-b65e-0e2b70290359)

Thanks for nay help.

**Comment by bdarcus** [comment]
It's because you're using a proportional font, and the UI depends on a monospace for alignment.

There may be ways for us to fix this so more general, but it would depend on a feature unavailable before Emacs 30 IIRC.

Bottom line for now: you want to change your font, at least for completion, to fix it.

**Comment by danielgoldin** [comment]
Ah, I thought as much. Any advice -- or example code -- for fixing just for completion.

**Reply by bdarcus** [reply]
[Looks like](https://emacs.stackexchange.com/questions/32615/change-spacemacs-default-font-except-minibuffer) you need to customize the `minibuffer-prompt` face.

**Comment by danielgoldin** [comment]
Thanks. That didn't work but I am happy with Inconsolata font as the default font.

On 2024-05-25 at 12:34 -07, bdarcus ***@***.***> wrote...
